modules/subscription/backend/modules/subscription/migrations/0001_initial.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 2.2.24 on 2022-03-13 02:52
```

Comment:
All the migrations should be squashed into one, but lets make sure that's the very last thing we do in the review process.